source/MaterialXView/Viewer.cpp

```diff
@@ -532,8 +532,18 @@ void Viewer::createSaveMaterialsInterface(Widget* parent, const std::string& lab
         if (!filename.empty() && !_materials.empty())
         {
             mx::DocumentPtr doc = _materials.front()->getDocument();
+            // Add element predicate to prune out writing elements from included files
+            auto skipXincludes = [this](mx::ConstElementPtr elem)
```

Comment:
Just an option, you can change the lambda scope from [this] to [&_xincludeFiles].